tests/Neo.Extensions.Tests/UT_StringExtensions.cs

```diff
@@ -42,5 +42,156 @@ public void TestGetVarSizeString()
             int result = "AA".GetVarSize();
             Assert.AreEqual(3, result);
         }
+
+        [TestMethod]
+        public void TestGetVarSizeInt()
+        {
+            for (int i = 0; i < 3; i++)
+            {
+                if (i == 0)
+                {
+                    int result = UnsafeData.GetVarSize(1);
+                    Assert.AreEqual(1, result);
+                }
+                else if (i == 1)
+                {
+                    int result = UnsafeData.GetVarSize(0xFFFF);
+                    Assert.AreEqual(3, result);
+                }
+                else
+                {
+                    int result = UnsafeData.GetVarSize(0xFFFFFF);
+                    Assert.AreEqual(5, result);
+                }
+            }
+        }
+
+        [TestMethod]
+        public void TestGetVarSizeGeneric()
+        {
+            for (int i = 0; i < 9; i++)
```

Comment:
?? why a for if i is not used in the test?

Reply:
not sure, i'm just moving the tests.